packages/dx-react-demos/src/bootstrap3/featured-controlled-mode/demo.jsx

```diff
@@ -159,30 +159,21 @@ export default class Demo extends React.PureComponent {
     this.changeCurrentPage = currentPage => this.setState({ currentPage });
     this.changePageSize = pageSize => this.setState({ pageSize });
     this.commitChanges = ({ added, changed, deleted }) => {
-      let rows = this.state.rows.slice();
+      let rows = this.state.rows;
       if (added) {
+        const startAddedId = (rows.length - 1) > 0 ? rows[rows.length - 1].id + 1 : 0;
```

Comment:
Consider starting this name with 'initial', 'first' or 'starting', not just 'start'.